src/setupTest.js

```diff
@@ -10,6 +10,8 @@ configure({ adapter: new Adapter() });
 
 process.env.LMS_BASE_URL = 'http://localhost:18000';
 process.env.MARKETING_SITE_BASE_URL = 'http://marketing.url';
+process.env.LOGOUT_URL = 'http://localhost:18000/logout';
```

Comment:
will we need to add this to the env in edx-internal?

Reply:
nope, this already exists https://github.com/edx/edx-internal/blob/7c59a075fe89248e6eed1da18e387bccab07e25b/frontends/common/prod_config.yml#L14

I just added it for the tests to have access to this ability